### ISSUE TITLE
Where do we document Adding to / Removing from Collections

### ISSUE THREAD

**Issue by azaroth42** [linked-art/linked.art#366]
Normally, the previous decisions about the default direction of a property make it obvious where references between entities should be managed. However in some cases, there are many occurrences on each side and thus no easy way to judge where best to record the knowledge.

In particular, I am updating the woefully out of date page on Collections. At the end is a section about the activities of adding an object to a collection or removing one from it.  Should these activities go on:

* The object (and thus there will be lots of adds/removes for each collection it could be part of)
* The collection (and thus there will be lots of adds/removes for each object that is part of it)
* A provenance activity (where there's fewer entities, but this doesn't capture all of the scenarios in which objects are added to collections ... for example a destruction event would surely cause an object to be removed, but doesn't generate a new provenance activity... or does it??)
* A new top level activity (which would be very small and seems to stand too much on its own)

I would suggest the object as a first attempt, as that's where we also have the `member_of` relationship to the set. 
Which subsequently means that as any entity can be part of a set, any entity can have the added/removed activity as part of its model.

Thoughts?

**Comment by azaroth42** [comment]
Similarly, we should discuss the Exhibition specific information for an object.

If it is associated with the Exhibition, then a big exhibition would have MANY attribute assignments for specific labels, descriptions, identifiers and so forth. However if it is associated with the object, then we decrease the ability to have information systems use each others' data directly ... the exhibiting institution cannot update the set of labels used in the owning institution's collection management system.

The same breakdown as above seems to apply:

* The object (lots of specific information in the record if it is in lots of exhibitions)
* The exhibition (lots of objects in the exhibition)
* A Provenance Activity (the transfer of custody to the exhibition, which might not be accurate and complex when the exhibition is carried out by the owning institution)
* The adding/removing of the object to the exhibition's set of objects (see above!!)
* New top level activity (that would be very small and easy to get lost)

For this one, I tend to prefer the object, as the number of exhibitions per object is smaller than the number of objects per exhibition.

**Comment by beaudet** [comment]
A cross-institutional ID for the exhibition seems important as well although that would probably be some type of activity grouping concept.  Would a convention need to be established to defer to the institution where the exhibition first opens to establish that ID or is there a need for some kind of independent authority to track exhibitions so that all of the participating institutions can point back to the same concept, e.g. Getty Exhibitions Authority?

**Comment by azaroth42** [comment]
Discussed on call 2020-10-21:  Should be in the collection, not on the object. See notes for rationale.

**Comment by azaroth42** [comment]
Pushing add/remove from Set, and join/leave from Group to 1.1

**Comment by azaroth42** [comment]
Useful for deaccessioned objects.

**Comment by azaroth42** [comment]
Consider also the joining/leaving of people/groups from groups. On the group or on the person?

**Comment by azaroth42** [comment]
Propose that it follows the "put the data on the child not the parent" strategy as it mirrors `member_of`. We don't have cross-institutional implementations yet, where an exhibition refers to the addition of another organization's object.

So on any entity, we would have link to the Addition which then refers to the Set that the entity was added to. Similarly Removal, and Joining/Leaving for Groups.

**Comment by azaroth42** [comment]
1.1 agreement, follow principles and goes on teh thing being added/removed

**Comment by Habennin** [comment]
So we have two collections that document this from earlier speculation:

addition to and removal from collection

We scope it to physical thing

Is what this is saying that we should have on every model aside from set (including set) the possibility of the event of adding the thing to a set or removing it? 

That would be a 1.1 thing?

https://pletka.io/projects/LA/collections/LAC.24

https://pletka.io/projects/LA/collections/LAC.23

**Comment by azaroth42** [comment]
Yes -- every model should have it, including Set. (We added this sub-collection to this parent collection)
And yes for 1.1 as it's not backwards incompatible.
